templates/support/URLSession+Api.swift.hbs

```diff
@@ -9,22 +9,40 @@ extension Foundation.URLSession {
     static var apiSession = Foundation.URLSession(configuration: .default)
 
     @available(*, renamed: "handleApiRequest(_:)")
-    static func handleApiRequest(_ request: Foundation.URLRequest, completion: @escaping ((_ response: Foundation.HTTPURLResponse?, _ data: Data?, _ error: Error?) -> Void)) {
+    static func handleApiRequest(_ request: Foundation.URLRequest, configuration: RetryConfiguration?, completion: @escaping ((_ response: Foundation.HTTPURLResponse?, _ data: Data?, _ error: Error?) -> Void)) {
```

Comment:
Let's rename this parameter from `configuration` to `retryConfiguration` so it doesn't look so weird everywhere.

OR

We could consider just passing it the whole configuration... retry is only one thing that we might like to configure about how we call our APIs. So pass the whole configuration and then let's use the retry piece out of it.

Reply:
Perhaps one day we'll need more config options. I think we should refactor it on that day. For now it is renamed.